archive/c/c/factorial.c

```diff
@@ -0,0 +1,69 @@
+#include <stdio.h>
+#include <stdint.h>
+#include <stdlib.h>
+#include <errno.h>
+#include <limits.h>
+
+int uint64_overflow(long, long);
+
```

Comment:
I think you should have the function `uint64_overflow()`  take two `uint64_t`s instead since the factorial result is a 64 bit integer.